docker-compose.override.standalone.yml

```diff
@@ -79,6 +79,19 @@ services:
     environment:
       STORAGES: ${STORAGES}
 
+  webdav:
```

Comment:
Don't forget to also update the qfield-docs architecture.md with the new changes once merged.